src/components/organisms/SiteFooter/index.tsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react'
-import GatsbyLink from '../../atoms/ThemedGatsbyLink'
 
 import { Box, Container, Flex, Heading } from '@theme-ui/components'
+import SiteLogo from '../../molecules/SiteLogo'
 import SmartLink from '../../atoms/SmartLink'
 
-import { ReactComponent as SiteLogo } from '@media/site-logo.svg'
 import { ReactComponent as DiscordIcon } from '@media/icons/discord.svg'
 import { ReactComponent as TwitterIcon } from '@media/icons/twitter.svg'
 import { ReactComponent as GithubIcon } from '@media/icons/github.svg'
 import { ReactComponent as YoutubeIcon } from '@media/icons/youtube.svg'
 import { ReactComponent as LinkedInIcon } from '@media/icons/linkedin.svg'
-import { ReactComponent as CmlIcon } from '@media/icons/cml.svg'
-import { ReactComponent as DvcIcon } from '@media/icons/dvc.svg'
-import { ReactComponent as StudioIcon } from '@media/icons/studio.svg'
-import { ReactComponent as IterativeIcon } from '@media/icons/iterative.svg'
+import { ReactComponent as IterativeIcon } from '@media/icons/iterative_icon-color--square_vector.svg'
+import dvcIcon from '@media/icons/dvc_icon-color--square_vector.svg'
+import cmlIcon from '@media/icons/cml_icon-color--square_vector.svg'
+import studioIcon from '@media/icons/studio_icon-color--square_vector.svg'
 
```

Comment:
not a fan of the naming convention...

- :heavy_check_mark: `@media/icons/github.svg`
- :question: `@media/icons/iterative_icon-color--square_vector.svg`

I'd suggest `@media/icons/iterative.svg`

only non-colour and/or non-square version(s) should have this mentioned in their names.

Reply:
yes and also rename the static repo ones to match :pray:

Reply:
Ok! I'll rename the ones for this repo, and talk to @yalozhkin about renaming the static repo images.

Reply:
I'm not following everything that is going on here in the code.  But we need the black and white versions available too. 
This is the image from ODSC West.  These conferences need to be readily able to get these images, not make them themselves.  
![image](https://user-images.githubusercontent.com/34289199/136054379-1a998c6a-63b5-4260-9314-2dd44208deff.png)

Reply:
@jendefig I guess that would be more about https://github.com/iterative/iterative.ai/issues/290 instead?

Reply:
File names updated!